meta/data.go

```diff
@@ -34,6 +36,15 @@ type Data struct {
 	MaxNodeID       uint64
 	MaxShardGroupID uint64
 	MaxShardID      uint64
+
+	Logger *log.Logger
+}
+
+// NewData returns a new instance of Data.
+func NewData() *Data {
+	return &Data{
+		Logger: log.New(os.Stderr, "[metadata] ", log.LstdFlags),
```

Comment:
At least, I think so, since we tend to name stuff after packages. Alternatively it should be name `data`. I'm not actually sure.

@corylanou -- what do you think?